### ISSUE TITLE
Rename issue

### ISSUE THREAD

**Issue by theironrobin** [flathub/com.github.alexr4535.siglo#17]
@jwillikers Do you know how i can go about renaming? from com.github.alexr4535.siglo to com.github.theironrobin.siglo? thanks.

**Comment by jwillikers** [comment]
@theironrobin Yes. What you need to do is described [here](https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life). You'll want to contact an admin per the instructions there and add a section to the `flathub.json` file to indicate that the repository has been renamed.

**Comment by theironrobin** [comment]
@jwillikers please review

**Comment by jwillikers** [comment]
Looks good to me. May want to wait until an admin responds to your request and the new repo is up before merging though.

**Comment by theironrobin** [comment]
reopened until an admin responds :)

**Comment by barthalion** [comment]
This currently points to an application which does not exist. If you want to rename it to `com.github.theironrobin.siglo`, you need to follow the submission procedure.

**Comment by theironrobin** [comment]
@barthalion submission added. please review: https://github.com/flathub/flathub/pull/3284

**Comment by theironrobin** [comment]
@barthalion can we remove this one from flathub now? the rebase is now published here: https://flathub.org/apps/details/com.github.theironrobin.siglo

**Comment by theironrobin** [comment]
any idea @jwillikers?

**Comment by jwillikers** [comment]
I think it needs to be left up for the redirect to continue. You should be able to just leave it, but maybe put a link to the new repository in the README to point to the right place. Perhaps in the future in the repository might be archived, though I'm not sure. The rediirect you added should continue to migrate everyone downloading the app to the new repository.

**Comment by theironrobin** [comment]
@jwillikers Do you know why auto-update doesn't seem to work to update siglo on flathub? (should be v0.9.9)
https://github.com/flathub/com.github.theironrobin.siglo/actions/runs/2614334071

**Comment by jwillikers** [comment]
> @jwillikers Do you know why auto-update doesn't seem to work to update siglo on flathub? (should be v0.9.9) https://github.com/flathub/com.github.theironrobin.siglo/actions/runs/2614334071

That Github Action actually only updates the dependencies and not Siglo itself. For that, you need to add the `x-checker-data` section back to your [new repository's Flatpak manifest](https://github.com/flathub/com.github.theironrobin.siglo/blob/master/com.github.theironrobin.siglo.json). The section exists in this repository's Flatpak manifest [here](https://github.com/flathub/com.github.alexr4535.siglo/blob/d2265c5e3cc72d8309a250709169324843654700/com.github.theironrobin.siglo.json#L38-L44).
